tests/test_nuclei_segmentation.py

```diff
@@ -118,3 +120,155 @@ def test_segment_nuclei_guassian_voting(self):
 
         assert len(nuclei.X) > 50
         assert len(votes) > 1000
+
+    # Testing arguments
+    args = argparse.Namespace(inputImageFile='',
+                              ImageInversionForm='Yes',
+                              analysis_mag=20.0,
+                              analysis_roi=[-1.0, -1.0, -1.0, -1.0],
+                              analysis_tile_size=1024.0,
+                              foreground_threshold=60.0,
+                              frame='0',
+                              ignore_border_nuclei=False,
+                              local_max_search_radius=10.0,
+                              max_radius=20.0,
+                              min_fgnd_frac=0.25,
+                              min_nucleus_area=80.0,
+                              min_radius=6.0,
+                              nuclei_annotation_format='boundary',
+                              num_threads_per_worker=1,
+                              num_workers=-1,
+                              reference_mu_lab=[8.63234435,
+                                                -0.11501964,
+                                                0.03868433],
+                              reference_std_lab=[0.57506023,
+                                                 0.10403329,
+                                                 0.01364062],
+                              scheduler='',
+                              stain_1='hematoxylin',
+                              stain_1_vector=[-1.0,
+                                              -1.0,
+                                              -1.0],
+                              stain_2='eosin',
+                              stain_2_vector=[-1.0,
+                                              -1.0,
+                                              -1.0],
+                              stain_3='null',
+                              stain_3_vector=[-1.0,
+                                              -1.0,
+                                              -1.0],
+                              style=None)
+
+    def test_image_inversion_flag_setter(self):
+        invert_image = nucl_det.image_inversion_flag_setter(self.args)
+        assert invert_image, True
```

Comment:
I think you want this to be 
```
invert_image,  default_img_inversion = nucl_det.image_inversion_flag_setter(self.args)
assert invert_image == True
assert default_img_inversion = False
```
As written, `invert_image` will be a tuple and assert checks that the (tuple, True) is not False, which isn't the test that was intended.

Reply:
Yeah, I agree. This test code was written when the function was only returning a boolean value. Then I added the second return default_img_inversion. I missed updating this test function also since it was giving me the test PASSED result.